app/src/main/res/layout/fragment_cocktail_details.xml

```diff
@@ -0,0 +1,187 @@
+                tools:text="Alcoholic" />
+
+            <TextView
+                android:id="@+id/label_category"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="10dp"
+                android:layout_marginTop="11dp"
+                android:background="#EA9922"
+                android:paddingHorizontal="7dp"
+                android:textSize="20sp"
+                app:layout_constraintStart_toEndOf="@id/icon_category"
+                app:layout_constraintTop_toBottomOf="@id/icon_alcoholic"
+                tools:text="Beer" />
+
+            <TextView
+                android:id="@+id/label_glass"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="15dp"
+                android:layout_marginTop="11dp"
+                android:background="#EA9922"
+                android:paddingHorizontal="7dp"
+                android:textSize="20sp"
+                app:layout_constraintStart_toEndOf="@id/icon_glass"
+                app:layout_constraintTop_toBottomOf="@id/icon_category"
+                tools:text="Beer Glass" />
+
+            <TextView
+                android:id="@+id/ingredients_label"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="20dp"
+                android:text="@string/ingredients"
+                android:textSize="25sp"
+                android:textStyle="bold"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/icon_glass" />
+
+            <TextView
+                android:id="@+id/ingredients_items"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="10dp"
+                android:textSize="20sp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/ingredients_label"
+                tools:text="Items" />
+
+            <TextView
+                android:id="@+id/instructions_label"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="20dp"
+                android:text="@string/instructions"
+                android:textSize="25sp"
+                android:textStyle="bold"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/ingredients_items" />
+
+            <TextView
+                android:id="@+id/instructions_items"
```

Comment:
Constraint this TV to the right side as well..we need to add some sort of margin on right side so the content is symmetrical.